README.md

```diff
@@ -75,9 +81,9 @@ Select 1:
 Identify the three places CSS can go, and rank them in terms of specificity:
 
 ```text
-1.
-2.
-3.
+1. Separate CSS stylesheet
+2. HTML code between <style></style> tags
```

Comment:
1 & 3 are correct! 

However, to be more specific for 2: 
It is within `<style></style>` tags in the HTML `<head>`, but we are still writing CSS code between them and not HTML code